en/assembly/quick_start_pixhawk4.md

```diff
@@ -46,7 +46,7 @@ Connect the output of the *Power Management Board* (PM board) that comes with th
 
 The image below shows the power management board provided with *Pixhawk 4*.
 
-![Pixhawk 4 - Power Management Board](../../assets/flight_controller/pixhawk4/pixhawk4_power_management_board.jpg)
+![Pixhawk 4 - Power Management Board](../../assets/flight_controller/pixhawk4/pixhawk4_power_management_board.png)

```

Comment:
The second diagram is much better, but way too big -649 KB. 

Wherever you have No 5V output, it might be better to label "Unpowered pins" or "Pins not powered"

Reply:
.... and we'd like to avoid that image being in the history so everyone has to download it.

Reply:
@hamishwillee can you check again? I compressed the diagram.

Reply:
Diagram better - but I was proposing removing "No 5V output" 0 - so it just says "unpowered pins".